ai/default/aitools.cpp

```diff
@@ -743,7 +743,6 @@ void dai_unit_new_task(struct ai_type *ait, struct unit *punit,
       if (unit_owner(missile) == unit_owner(punit)
           && def_ai_unit_data(missile, ait)->task != AIUNIT_ESCORT
           && !unit_transported(missile)
-          && unit_owner(missile) == unit_owner(punit)
```

Comment:
This does not seem to be related to the build tools update

Reply:
It's the same condition as in the first line